brands/shop/convenience.json

```
@@ -1,4 +1,15 @@
 {
+  "shop/convenience|Huck Food Fuel": {
+    "countryCodes": ["us"],
+    "matchNames": ["Huck's"],
+    "tags": {
+      "brand": "Huck Food Fuel",
+      "brand:wikidata": "Q56276328",
+      "brand:wikipedia": "en:Huck_Food_Fuel",
```

Looks like this should actually point at..
```suggestion
      "brand:wikipedia": "en:Huck's Food & Fuel",
```